_Group Projects/Group_2/Week_5/Readme.md

```diff
@@ -0,0 +1,21 @@
+Please change the working directory to the file path that goes to the Box folder on your own computer when running the demonstration do file
+
+This program will help you determine if you should run a linear regression model on your dataset, as well as let you know of any outliers.
+
+
+After you loaded the dataset, the syntax is 
+
+ . myprogram dep_var ind_var_1 ... ind_var n
```

Comment:
It's really neat that your program allows you to list many independent variables, as opposed to strictly requiring a certain number of them.